Sources/FeaturevisorSDK/Instance+Fetch.swift

```diff
@@ -0,0 +1,42 @@
+import FeaturevisorTypes
```

Comment:
do we wish to keep `+` in the filename? is it common in Swift?

Reply:
To my knowledge is a style thing inherited from Objective-C right? but I'm not the expert here nor I care that much about it

Reply:
Yes, it's a common thing for Swift world. In that way, we are pointing out that we have an extension (in this case to `FeaturevisorInstance`) and closing some context of usage here (for our case logic to fetch).